float32 steer_angle
float32 cur_dist
float32 tar_dist
float32 tar_vel
bool fi_encoder
bool alpha
bool beta
bool gamma
